cmd/generate/flag.go

```diff
@@ -32,13 +36,14 @@ func (f *flag) Init(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&f.ConfigVersion, flagConfigVersion, "", `Major part of the configuration version to use for generation (e.g. "v2").`)
 	cmd.Flags().StringVar(&f.Installation, flagInstallation, "", `Installation codename (e.g. "gauss").`)
 	cmd.Flags().StringVar(&f.GitHubToken, flagGithubToken, "", fmt.Sprintf(`GitHub token to use for "opsctl create vaultconfig" calls. Defaults to the value of %s env var.`, envConfigControllerGithubToken))
+	cmd.Flags().BoolVar(&f.LocalGenerator, flagLocalGenerator, false, `Use local filesystem as source of configuration.`)
 }
 
 func (f *flag) Validate() error {
 	if f.App == "" {
 		return microerror.Maskf(invalidFlagError, "--%s must not be empty", flagApp)
 	}
-	if f.ConfigVersion == "" && f.Branch == "" {
+	if f.ConfigVersion == "" && f.Branch == "" && !f.LocalGenerator {
```

Comment:
We need to handle all cases. 
- `f.ConfigVersion == "" && f.Branch` 
- `f.ConfigVersion == "" && f. LocalGenerator` 
- `f. Branch == "" && f. LocalGenerator`